Jarvis plugin is broken in latest pre-release

No error in the console but the setting panel and other options are gone.

Only message in console is this:

> 09:45:36: PluginService: Loading plugin from /joplin-desktop/cache/joplin.plugin.alondmnt.jarvis

It works in dev however, so there's a chance it's related to the new app packaging.

This seems to be the full error:
> ```
> Uncaught Error: Cannot find module '../../node_modules/@joplin/lib/node_modules/sqlite3/lib/sqlite3.js'
> Require stack:
> - /tmp/.mount_JoplindAyjmN/resources/app.asar/services/plugins/plugin_index.html
> ```

--- a/packages/app-desktop/services/plugins/plugin_index.js
+++ b/packages/app-desktop/services/plugins/plugin_index.js
@@ -51,10 +51,8 @@
 			const modulePath = args && args.length ? args[0] : null;
 			if (!modulePath) throw new Error('No module path specified on `require` call');
 
-			// The sqlite3 is actually part of the lib package so we need to do
-			// something convoluted to get it working.
 			if (modulePath === 'sqlite3') {
-				return require('../../node_modules/@joplin/lib/node_modules/sqlite3/lib/sqlite3.js');
+				return require('sqlite3');
 			}
 
 			if (modulePath === 'fs-extra') {

